news/2023/2023-06-17-new-featured-artist-junk.md

```diff
@@ -65,9 +66,9 @@ Finish things off with [this... *elegant*... beatmap](https://osu.ppy.sh/beatmap
 
 There's more **Junk** to go around. Check out [their Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/365) for the full tracklist, including more insane tracks for your avant-garde mapping escapades.
 
-In case you missed it last week, we announced some [new **Featured Artist playlists** for osu!(lazer)](https://osu.ppy.sh/home/news/2023-06-03-new-featured-artist-andora#featured-artist-playlists)! If you're looking to play some cool new maps (or you just want an excuse to try out the latest lazer update), this is for you.
+In case you missed it two weeks ago, we announced some [new **Featured Artist playlists** for osu!(lazer)](https://osu.ppy.sh/home/news/2023-06-03-new-featured-artist-andora#featured-artist-playlists)! If you're looking to play some cool new maps (or just want an excuse to try out the latest lazer update), this is for you.
 
-And that's all for now! Another artist is on its way next week, so track the [@osugame Twitter](https://twitter.com/osugame) or the `#featured-artists` channel in the [osu! Discord server](https://discord.gg/ppy) for all the news on that.
```

Comment:
what's with this url change..?

Reply:
discord uses discord.com nowadays, so this prevents a redirect